enemy_tools.py

```diff
@@ -0,0 +1,34 @@
+import enemy
+import utils
+
+def get_enemy_size_from_type(enemy_type):
+    return utils.get_enemy_size(enemy_type)
+    
+
+# Since the stage is what determines the enemy's type, let's isolate that into a simple function
+def get_enemy_type_from_stage(stage):
+    match stage:
+        case 1:
+            return 'enemy1'
+        case 2:
+            return 'enemy2'
+        case 3:
+            return 'boss3'
+        case 4:
+            return 'boss4'
+        case 5:
+            return 'boss5'
```

Comment:
This "match/case" structure is a more elegant way to handle this type of thing, instead of having a bunch of if/elif/elif/elif/elifs in a row.